arch/lkl/kernel/setup.c

```diff
@@ -112,7 +112,8 @@ void machine_restart(char *unused)
 long lkl_sys_halt(void)
 {
 	long err;
-	long params[6] = { 0, };
+	long params[6] = {LINUX_REBOOT_MAGIC1,
+		LINUX_REBOOT_MAGIC2, LINUX_REBOOT_CMD_RESTART, };
```

Comment:
Not a big deal, but we probably want to use LINUX_REBOOT_CMD_HALT instead, just to match the API name.